adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -26,19 +26,29 @@ const translated = {
   share: django.gettext('Share'),
   report: django.gettext(' Report'),
   showModStatement: django.gettext('Show moderator\'s feedback'),
-  hideModStatement: django.gettext('Hide moderator\'s feedback')
+  hideModStatement: django.gettext('Hide moderator\'s feedback'),
+  deleteComment: django.gettext('Do you really want to delete this comment?'),
+  delete: django.gettext('Delete'),
+  abort: django.gettext('Abort'),
+  deletedyByCreatorOn: django.gettext('Deleted by creator on'),
+  deletedByModeratorOn: django.gettext('Deleted by moderator on'),
+  blockedByModeratorOn: django.gettext('Blocked by moderator on'),
+  lastEditOn: django.gettext('Latest edit on'),
+  moderator: django.gettext('Moderator'),
+  hideReplies: django.gettext('hide replies'),
+  reply: django.pgettext('verb', 'Reply')
```

Comment:
I pulled out all the translations into this because I thought we decided that translations should always be on top but then realized there are a lot of files which don't have the `translated` at all, so I didn't continue as this would have taken quite long

Reply:
Yes, the translations need to be on top not to get lost in the makemessages. 
I think, we wanted to change that when we touch the files, but we might have forgotten again.

Reply:
how do we handle plurals? as they depend on a variable they can't be in the `translated` block? If we get rid of all template strings I guess that's not a problem anymore?